Mathlib/Tactic/Linter/Common.lean

```diff
@@ -0,0 +1,36 @@
+/-
+Copyright (c) 2024 Moritz Firsching All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa, Moritz Firsching
+-/
+
+import Lean.Elab.Command
+open Lean Parser Elab Command Meta
```

Comment:
The style linter complains: you should put the module docstring before any open statements.